.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "traildb"]
+	path = traildb
+	url = https://github.com/traildb/traildb.git
```

Comment:
sure you want this to be a submodule? seems rather unintuitive.

Reply:
At first I thought the same about using git submodules, but after looking at other crates I have found that mainly two approaches are used to deal with this:
- use git submodules
- commit C sources (or tar with sources) to git repository

When using the 1st approach, users of `traildb-rust` won't even notice that git submodules are used. The cargo will checkout git submodule automatically during the build process. Only downside of this approach is that the developers of `traildb-rust` will have to run `git submodule update --init --recursive` if they need to build traildb library from sources. If they have traildb already installed as a system library, then they don't have to do it at all.

Reply:
I have moved traildb build into a separate package + traildb sources are now part of the repo.